cdk/lib/lb.ts

```diff
@@ -78,7 +86,7 @@ export class CacclLoadBalancer extends Construct {
       protocol: ApplicationProtocol.HTTPS,
       // setting this duration value enables the lb stickiness; 1 day is the default
       stickinessCookieDuration: Duration.seconds(86400),
-      deregistrationDelay: Duration.seconds(30),
+      deregistrationDelay: Duration.seconds(targetDeregistrationDelay),
```

Comment:
So it was 30 seconds originally, but now it defaults to 30 but is configurable?

Reply:
Correct. This was something @gabeabrams asked for a long time ago.